src/XMakeBuildEngine/UnitTests/Evaluation/ProjectRootElementCache_Tests.cs

```diff
@@ -189,5 +189,61 @@ public void GetProjectRootElementChangedOnDisk2()
                 File.Delete(path);
             }
         }
+
+        /// <summary>
+        /// This test replicates the scenario in https://devdiv.visualstudio.com/DevDiv/_workitems?id=366077&_a=edit
```

Comment:
GitHub highlights an illegal ampersand here. I think you can just cut off that part of the URL and it'll still work.